### ISSUE TITLE
Allow posting of Sentry error messages in Status App, on occurrence of error (summarised error)

### ISSUE THREAD

**Issue by harmeet-status** [sartography/spiff-arena#779]
When a user experiences an error, they most likely will need to contact support. However the process of screenshotting or remembering the error is hard and usually when the user gets in touch with support, they often can barely remember the error etc.

In order to make this easier, we want to do the following:
1. Errors are captured by Sentry and we want to post those to a status channel
2. Ideally we would also like to tag in the user who experienced this error in the channel. This should be possible since the Spiff Waku node will have contact request approved.

**Comment by harmeet-status** [comment]
Would this be sending to a channel within a Discord Server?

**Comment by burnettk** [comment]
Yeah, it could be any channel.

**Comment by harmeet-status** [comment]
Yeah lets do this in Discord for now, will migrate over to Status App once we get Status Communities working well.
